### PR TITLE
DRILL-6204: Pass tables columns without partition columns to empty Hi…

### PR DESCRIPTION
**Pull request by arina-ielchiieva**
…ve reader

Details in [DRILL-6204](https://issues.apache.org/jira/browse/DRILL-6204).

### CODE REVIEW

**Comment by arina-ielchiieva** [comment]
@parthchandra, @vdiravka please review.

File: contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveDrillNativeScanBatchCreator.java [hunk 1]
```diff
@@ -174,7 +174,7 @@ public ScanBatch getBatch(ExecutorFragmentContext context, HiveDrillNativeParque
     // If there are no readers created (which is possible when the table is empty or no row groups are matched),
     // create an empty RecordReader to output the schema
     if (readers.size() == 0) {
-      readers.add(new HiveDefaultReader(table, null, null, columns, context, conf,
+      readers.add(new HiveDefaultReader(table, null, null, newColumns, context, conf,
```
Comment (vdiravka) [inline contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveDrillNativeScanBatchCreator.java]: Could we rename newColumns -> nonPartitionedColumns or tableColumns?

**Comment by arina-ielchiieva** [comment]
@vdiravka, thanks for the code review. Updated PR.

**Comment by parthchandra** [comment]
+1

**Pull request closed without merging**